src/analysis/analysis-model.js

```diff
@@ -68,6 +70,16 @@ var AnalysisModel = Model.extend({
     _.each(this.getParamNames(), function (paramName) {
       this.bind('change:' + paramName, this._reloadVis, this);
     }, this);
+
+    this.bind('change:status', function () {
+      if (this._hadStatus() && this.isReady() && this.isSourceOfAnyModel()) {
```

Comment:
I suppose _hadStatus checks for a previous status, can an analysis start in "ready" status?

Reply:
Analysis models don't have a `status` until they get one from Maps API.